tests/tools/extract-tools-github.test.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
+const { getData } = require('../../scripts/tools/extract-tools-github');
+
+jest.mock('axios');
+
+describe('getData', () => {
+  it('should return data when API call is successful', async () => {
+    const mockData = {
+      data: {
+        name: '.asyncapi-tool',
+        path: 'asyncapi/.asyncapi-tool',
+      },
+    };
+
+    axios.get.mockResolvedValue(mockData);
+
+    const result = await getData();
+
+    expect(result).toEqual(mockData.data);
+    expect(axios.get).toHaveBeenCalledWith(
+      'https://api.github.com/search/code?q=filename:.asyncapi-tool',
+      {
+        headers: {
+          accept: 'application/vnd.github.text-match+json',
```

Comment:
@vishvamsinh28 can you please explain this part

Reply:
It is to verify whether the API is called with the correct URL and headers as specified in the script.